pkg/ui/workspaces/cluster-ui/src/databaseDetailsPage/databaseDetailsPage.tsx

```diff
@@ -669,13 +672,16 @@ export class DatabaseDetailsPage extends React.Component<
               Table Stats Last Updated <Timezone />
             </Tooltip>
           ),
-          cell: table => (
-            <Timestamp
-              time={table.details.statsLastUpdated?.stats_last_created_at}
-              format={DATE_FORMAT}
-              fallback={"No table statistics found"}
-            />
-          ),
+          cell: table =>
+            checkInfoAvailable(
```

Comment:
It looks like `checkInfoAvailable` should be written/used as a component. You could probably rewrite it as a container and then just put the `cell` parameter as a child. I think this would follow React's principle of encouraging composition patterns. Don't need to address this in this PR, but just something to consider for future iterations!